.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on:
+  pull_request:
+  push:
+  release:
+    types: published
+
+jobs:
+  build:
+    runs-on: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install dependencies
```

Comment:
consider inserting here:

```yml
    - uses: actions/setup-python@v1
      with:
        python-version: '3.x'
```

to use python in Actions in a friendlier/easier/faster way

Reply:
Why would that be faster?

Reply:
Instead of `apt install` out of date python packages, "uses python" is instant, and enables `pip install` which is typically faster and gives the latest version (or allows selecting a specific version)

Reply:
Not using "uses python" is also instant, since ubuntu-latest ships Python 3 ;)